Decode/PerfItemValue.cs

```diff
@@ -629,83 +654,250 @@ public string GetString()
                     Value128Append(sb, 0);
                     break;
                 case EventHeaderFieldEncoding.ZStringChar8:
+                    this.StringAppend(sb, convertOptions, Text.Encoding.UTF8);
+                    break;
+                case EventHeaderFieldEncoding.ZStringChar16:
+                case EventHeaderFieldEncoding.StringLength16Char16:
+                    this.StringAppend(
+                        sb,
+                        convertOptions,
+                        this.ByteReader.FromBigEndian ? Text.Encoding.BigEndianUnicode : Text.Encoding.Unicode);
+                    break;
+                case EventHeaderFieldEncoding.ZStringChar32:
+                case EventHeaderFieldEncoding.StringLength16Char32:
+                    this.StringAppend(
+                        sb,
+                        convertOptions,
+                        this.ByteReader.FromBigEndian ? PerfConvert.EncodingUTF32BE : Text.Encoding.UTF32);
+                    break;
+                case EventHeaderFieldEncoding.BinaryLength16Char8:
                 case EventHeaderFieldEncoding.StringLength16Char8:
```

Comment:
Note: New `BinaryLength16Char8` encoding. Semantics are identical to `StringLength16Char8` except that the default format is `HexBytes` instead of `StringUtf`.